### DISCUSSION TITLE
求助，怎么查看supadata的信息

### DISCUSSION THREAD

**Discussion by Dr-WillMos** [Qexo/Qexo#567] [Q&A] [answered]
![图片](https://github.com/user-attachments/assets/f1cf56b2-5a78-40ac-a2e2-90fe2f07671f)
这一条在哪里看呢，我的host是一个以aws开头的。
使用Vercel数据库部署的时候出现
![图片](https://github.com/user-attachments/assets/bc8b301d-d173-40aa-8af0-c13c2635980d)

**Comment by am-abudu** [comment]
请点击expand

**Reply by Dr-WillMos** [reply]
![图片](https://github.com/user-attachments/assets/9a6947b9-5282-4807-9b4b-5a6af97ada2e)
![图片](https://github.com/user-attachments/assets/2a68811b-38f5-42d8-a62b-2b07de5f2967)
vercel的数据库部署出现的问题。

**Reply by am-abudu** [reply]
请仔细阅读文档![Screenshot_2024-08-26-20-08-57-40_a252b927494330cdc2c8ba3b3f952e5e.png](https://github.com/user-attachments/assets/0ab3f41d-459e-46e4-a8be-a5172fda9b52)

**Reply by Dr-WillMos** [reply]
抱歉，但在更改了版本之后出现了以下报错
![图片](https://github.com/user-attachments/assets/59f3d52c-a85e-485b-be97-e9521bcaacf3)
但在部署之前数据库显示已经链接
![图片](https://github.com/user-attachments/assets/9f27d969-35cc-4817-8a7b-442861710954)

**Comment by am-abudu** [comment]
请查看项目Environment Variables是否存在`POSTGRES_`开头的字段

**Reply by Dr-WillMos** [reply]
![图片](https://github.com/user-attachments/assets/c10a36d8-3c6e-4d64-abd4-39b19f1b4fd9)
如图

**Reply by am-abudu** [reply]
> ![图片](https://github.com/user-attachments/assets/c10a36d8-3c6e-4d64-abd4-39b19f1b4fd9)
> 如图

这是正确的，你可以尝试重新部署

**Reply by Dr-WillMos** [reply]
![图片](https://github.com/user-attachments/assets/206c115e-9ea5-4f94-bfb4-409edec31d6a)
![图片](https://github.com/user-attachments/assets/276c31d5-d1c1-42d5-ae6c-300f74c2070f)
已经尝试重新部署，但仍然报错。

**Reply by am-abudu** [reply, marked as answer]
你没有为Environment Variables 设置Production权限。表现为显示的是Preview, Development而没有Production

**Reply by Dr-WillMos** [reply]
我需要怎么把已有的环境变量设置为Production权限，该环境变量似乎无法删除

**Reply by am-abudu** [reply]
你需要删除connect 然后在重新创建时选择

**Reply by Dr-WillMos** [reply]
已经成功，感谢您的指导。
